### PR TITLE
LiveChat integration docs

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for LiveChat.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/livechat.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+<div class="infoBlock">
+RudderStack only tracks the CTA interactions specified in the <strong>List of Events</strong> field in the dashboard settings. If this setting is not specified, RudderStack tracks all the interactions specified in the <Link to="#supported-events">Supported events</Link> section below.
+</div>
+
+### Supported events
+
+RudderStack automatically records and sends the following <a href="https://developers.livechat.com/docs/extending-chat-widget/javascript-api#callbacks">LiveChat callbacks</a>:
+
+| LiveChat event | `track` event name | Description |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/livechat.mdx:84]: @RohitKhatta @shrouti1507 we only need to add the corresponding `track` call names in the below table corresponding to the LiveChat callback events.

File: docs/destinations/streaming-destinations/livechat.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+Once you have confirmed that the source platform supports sending events to LiveChat, follow these steps:
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **LiveChat**.
+2. Assign a name to the destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure LiveChat as a destination, you will need to configure the following settings:
+
+<img src="../../assets/event-stream-destinations/livechat-connection-settings.png" alt="LiveChat connection settings" />
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/livechat.mdx:33]: @RohitKhatta please provide the connection settings image.

**Comment by ameypv-rudder** [comment]
@shrouti1507 @RohitKhatta the doc is ready for your review.

File: docs/destinations/streaming-destinations/livechat.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **LiveChat**.
+2. Assign a name to the destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure LiveChat as a destination, you will need to configure the following settings:
+
+<img src="../../assets/event-stream-destinations/livechat-connection-settings-1.png" alt="LiveChat connection settings" />
+
+- **License**: Enter your LiveChat license ID.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/livechat.mdx:35]:
```suggestion
- **License ID**: Enter your LiveChat license ID.
```

File: docs/destinations/streaming-destinations/livechat.mdx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  - **List of Events**: If **Record Live Chat Events** setting is enabled, enter the list of LiveChat interactions for RudderStack to track.
+  - **Update Event Name in track call**: Use this setting to map the standard LiveChat callback events with custom event names.
+
+  <img src="../../assets/event-stream-destinations/livechat-connection-settings-3.png" alt="LiveChat connection settings" />
+
+  <div class="infoBlock">
+  For more information on this setting, refer to the <Link to="#mapping-events">Mapping events</Link> section below.
+  </div>
+
+- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to ConvertFlow.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/livechat.mdx:54]:
```suggestion
- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to LiveChat.
```

File: docs/destinations/streaming-destinations/livechat.mdx [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+</div>
+
+- **Use device mode to send events**: As this is a <Link to="https://www.rudderstack.com/docs/destinations/rudderstack-connection-modes/#device-mode">web device mode</Link>-only destination, this setting is enabled by default and cannot be disabled.
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> events to send all the user-related information to LiveChat.
+
+RudderStack sends the various **optional** `identify` properties by calling the following LiveChat functions:
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/livechat.mdx:68]:
```suggestion
RudderStack sends various **optional** `identify` properties by calling the following LiveChat functions:
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**